daemon/pod.go

```diff
@@ -285,14 +286,15 @@ func NewPod(podSpec *apitypes.UserPod, id string, data interface{}) (*Pod, error
 
 	p := &Pod{
 		Id:         id,
+		CreatedAt:  time.Now().Unix(),
```

Comment:
it might be a different value when it is call from `dameon.Restore() -> daemon.createPodInternal() -> .... `